ext/jenkins/beaker-tests.sh

```diff
@@ -24,19 +24,12 @@ export BEAKER_PRESERVE_HOSTS=onfail
 export PUPPETDB_PACKAGE_REPO_HOST="puppetdb-prerelease.s3.amazonaws.com"
 export PUPPETDB_PACKAGE_REPO_URL="http://puppetdb-prerelease.s3.amazonaws.com/puppetdb/${PACKAGE_BUILD_VERSION}"
 export PUPPETDB_PACKAGE_BUILD_VERSION=$PACKAGE_BUILD_VERSION
-
-# Setup RVM and gemset
-[[ -s "/usr/local/rvm/scripts/rvm" ]] && source /usr/local/rvm/scripts/rvm
-rvm use ruby-2.0.0-p481
-
-# Remove old vendor directory to ensure we have a clean slate
-if [ -d "vendor" ];
-then
-  rm -rf vendor
-fi
-mkdir vendor
-
-bundle install --path=vendor/bundle --without=test --retry=10
+export BEAKER_CONFIG=$LAYOUT
 
 # Now run our tests
 bundle exec rake beaker:acceptance
+exit_code=$?
+
+$WORKSPACE/ext/jenkins/cleanup-s3.sh
```

Comment:
Ah, good point. I was thinking about other reasons to keep this a separate job anyway yesterday (we probably want this step to run regardless of whether some command earlier in the `beaker-tests.sh` script fails). 

I will update puppetlabs/ci-job-configs#469 with a new job to run this command and remove this line.